packages/cubejs-client-vue/src/QueryRenderer.js

```diff
@@ -74,6 +74,8 @@ export default {
       slot = $scopedSlots.error ? $scopedSlots.error({ error, sqlQuery }) : slot;
     }
 
+    slot= ($scopedSlots.error && this.builderProps.metaError)?$scopedSlots.error({metaError:this.builderProps.metaError}):slot;
```

Comment:
I think it would be better to create a `query-builder` error slot and handle a possible `.meta()` error there. This way we'll decouple `query builder` and `query execution` error handling.